client.go

```diff
@@ -39,6 +40,10 @@ import (
 	"github.com/hazelcast/hazelcast-go-client/types"
 )
 
+const (
+	ClientVersion = internal.ClientVersion
```

Comment:
ide says unused? Is this a leftover?

Reply:
No, it's not a leftover. I'll add a fake test so it doesn't cause any linter warnings.